frontend/src/components/header/Header.tsx

```diff
@@ -82,6 +82,9 @@ export default function Header() {
                     >
                         Shop
                     </HeaderLink>
+                    <HeaderLink href="https://navoneedankje.nl">
+                        Anti-NAVO-Campagne
```

Comment:
```suggestion
                        Anti-NAVO Campagne
```

Reply:
Mij werd precies geïnstrueerd om het zo te schrijven door meerdere bestuursleden

Reply:
Dubbele hyphen lijkt me spelling technisch gezien altijd onjuist. En Anti-NAVO is hier een bijvoeglijk naamwoord voor het zelfstandig naamwoord campagne, dus daarom denk ik dat daar een spatie tussen mag.

Dat gezegd hebbende, ik heb totaal geen gevoel voor taal dus mogelijk is wat ik zeg onzin. So feel free to merge as is.